README.md

```diff
@@ -363,6 +363,30 @@ const options = {
 }
 ```
 
+## Collaboration
```

Comment:
How about calling this `Contributing` instead, as that's the more common title for this kind of section.